generators/app/tasks/appSetup/enablePushNotifications.js

```diff
@@ -1,17 +0,0 @@
-module.exports = function enablePushNotifications() {
```

Comment:
Comment that this file is deprecated (and hence is being deleted) in this PR's change log please :)